src/fake_useragent/utils.py

```diff
@@ -80,17 +43,13 @@ def load() -> list[BrowserUserAgentData]:
         list[BrowserUserAgentData]: The list of browser user agent data, following the
             `BrowserUserAgentData` schema.
     """
-    data = []
     try:
-        json_path = find_browser_json_path()
-        for line in json_path.read_text().splitlines():
-            data.append(json.loads(line))
+        jsonl = get_data(__package__ or "fake_useragent", "data/browsers.jsonl")
```

Comment:
Why is this `or` necessary? Might warrant a comment, as it's unexpected to me.

Reply:
It's because `__package__` can be `None`. Pylance told me that its type is `str | None`.

It seems `__package__` will be `None` only when this file is executed directly: https://peps.python.org/pep-0366/#proposed-change

So maybe I can insert one line of `assert __package__ is not None` before this line to ensure type safety?

Reply:
Yeah, if the only `None` case is running the file directly, it makes no sense to guard for it IMO. I would either:
1. `typing.cast` solve it.
2. Add an `if`-`else` with a `raise`.
3. Do the `assert` (which is basically an `if`-`else` that's less pretty).

Reply:
4. Just use the literal string "fake_useragent"

Reply:
At that point, if we're going to hardcode the string only for a fringe case that should never happen, we're better off just using the hardcoded string directly. My initial suggestion had the `__package__` variable cause that's what I saw used with `get_data`, but I'm unfamiliar with this.

Again, not blocking the PR due to any of this!